src/scss/components/switch/_index.scss

```diff
@@ -39,10 +39,12 @@ hx-switch-control {
 hx-switch-control > input[type="checkbox"] ~ label {
   > hx-switch {
     background-color: $gray-0;
+    background-color: var(--hxSwitchControl-switch-bgcolor, $gray-0);
     border: 0.125rem solid $cyan-900;
```

Comment:
Please use updated colors to replace `$cyan-900`.

Reply:
Changes have been done, please review.